lib/FormCustomProperties/CustomPropertiesListField.js

```diff
@@ -216,18 +198,51 @@ export default class CustomPropertiesListField extends React.Component {
       });
     };
 
-    return (
-      <FieldComponent
-        data-test-customproperty-value
-        error={!isEmpty(errors) ? errors[customProperty.value] : undefined}
-        id={`edit-customproperty-${i}-value`}
-        label={<FormattedMessage id="stripes-erm-components.prop.customPropertyValue" />}
-        onChange={handleChange}
-        value={controlledFieldValue}
-        {...fieldProps}
-        required={!customProperty.primary}
-      />
-    );
+    return (customProperty.type === customPropertyTypes.NUMBER || customProperty.type === customPropertyTypes.DECIMAL) ?
+      (
+        <IntlConsumer>
+          { intl => (
+            <Field
+              badInput={intl.formatMessage({ id: 'stripes-erm-components.errors.customPropertyInvalidNumber' })}
+              component={TextField}
+              data-test-customproperty-value
+              inputRef={this.inputRef}
```

Comment:
This `inputRef` isn't doing anything anymore is it?

Reply:
No, it isnt.